content/updates/2017-05-22-release-notes.md

```diff
@@ -0,0 +1,48 @@
++++
+date = "2017-05-22"
+title = "Platform Release Notes: May 22, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+### Researching
```

Comment:
I know it doesn't fit with the style of the other headings but renaming this to "Volunteers needed" or something like that it will help highlight the call to action.